.github/workflows/constraints.txt

```diff
@@ -0,0 +1,3 @@
+pip==23.0
```

Comment:
Cool, this should be covered by dependabot right now too. So this should start getting auto prs to get updated as well.